src/renderer/components/+custom-resources/__tests__/custom-resource-details.test.tsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import React from "react";
+import { CustomResourceDefinition } from "../../../../common/k8s-api/endpoints";
+import { KubeObject } from "../../../../common/k8s-api/kube-object";
+import { getDiForUnitTesting } from "../../../getDiForUnitTesting";
+import type { DiRender } from "../../test-utils/renderFor";
+import { renderFor } from "../../test-utils/renderFor";
+import { CustomResourceDetails } from "../crd-resource-details";
+
+describe("<CustomResourceDetails />", () => {
+  let render: DiRender;
+
+  beforeEach(() => {
+    const di = getDiForUnitTesting({ doGeneralOverrides: true });
+
+    render = renderFor(di);
+  });
+
+  describe("with a CRD with a boolean field", () => {
+    const crd = new CustomResourceDefinition({
```

Comment:
This `crd` should be in `beforeEach`. It's now shared between tests. Just to limit possibility of one test affecting other test.

Reply:
Sounds good